addons/binding/org.openhab.binding.netatmo/README.md

```diff
@@ -92,6 +92,10 @@ Number Netatmo_Indoor_CO2 "CO2" <carbondioxide> { channel = "netatmo:NAMain:home
 * Location
 * TimeStamp
 * LastStatusStore
+* MinTemp
+* MaxTemp
```

Comment:
I'd like but these values are not documented in Netatmo API, so observation may help to understand them